src/query/auth.ts

```diff
@@ -29,4 +29,28 @@ const useLogin = () => {
   return { login: mutate, error, isError };
 };
 
-export default useLogin;
+export const useVerifyEmail = (emailInput: string, setIsSendAuth: (value:boolean)=>void) => {
+  const { mutate, isLoading, isSuccess } = useMutation({
+    mutationFn: () => findPasswordVerify({ email: emailInput }),
+    onSuccess: () => {
+      setIsSendAuth(true);
+    },
+  });
+  return {
+    verifyEmail: mutate, mutate, isLoading, isSuccess,
+  };
+};
+
+export const useSubmit = (emailInput: string, verifyInput:string) => {
+  const navigate = useNavigate();
+  const { mutate } = useMutation({
+    mutationFn: () => findPassword({ address: emailInput, certificationCode: verifyInput }),
```

Comment:
위 구현은 간단히 이렇게 할 수 있어여
```suggestion
export const useSubmit = () => {
  const navigate = useNavigate();
  const { mutate } = useMutation({
    mutationFn: (emailInput: string, verifyInput:string) => findPassword({ address: emailInput, certificationCode: verifyInput }),
```

Reply:
Hook이 상태에 의존하지 않도록 수정했습니다. 2e7495dd8f36faf7a2b17b8d981d11a41fd916a7